rancher/v1.2/en/quick-start-guide/index.md

```diff
@@ -79,7 +79,7 @@ If you click on the dropdown of the **_first-container_**, you will be able to p
 Rancher will display any containers on the host even if the container is created outside of the UI. Create a container in the host's shell terminal.
 
 ```bash
-$ docker run -d -it --name=second-container ubuntu:14.04.2
+$ docker run -it --name=second-container ubuntu:14.04.2
```

Comment:
Exactly, the guide says right after that:

> In the UI, you will see second-container pop up on your host! If you terminate the container by exiting the shell, the Rancher UI will immediately show the stopped state of the container.

With the `-d`, if you exit the shell the container won't stop, and the Rancher UI won't show the stopped state of the container.